dll/win32/kernel32/client/path.c

```diff
@@ -1114,6 +1115,26 @@ GetFullPathNameW(IN LPCWSTR lpFileName,
                                 lpFilePart) / sizeof(WCHAR);
 }
 
+/*
+ * @implemented
+ */
+BOOL
+WINAPI
+SetSearchPathMode(DWORD dwFlags)
+{
+    NTSTATUS Status;
+
+    /* Call Rtl to do the work */
+    Status = RtlSetSearchPathMode(dwFlags);
+    if (!NT_SUCCESS(Status))
+    {
+        SetLastError(RtlNtStatusToDosError(Status));
+        return FALSE;
+    }
+
+    return NT_SUCCESS(Status);
```

Comment:
```suggestion
    return TRUE;
```

Reply:
Is this correct? Shouldn't this function receive the appropriate status from the actual Rtl one instead? :thinking:

Reply:
Sorry, I don't understand what you are asking for.

Reply:
@oleg-dubinskiy : Since you've handled the case where `NT_SUCCESS(Status)` is FALSE above (and you unconditionally return FALSE in that case), then now when the code flow is reaching that `return` statement, `NT_SUCCESS(Status)` can only be TRUE. That's why Serge suggested this change.

Reply:
OK, thanks @HBelusca. Then this is indeed correct. Sorry if that.

Reply:
Committed.